### DISCUSSION TITLE
what is the best way of using windows / chromeos dualboot on pixelbook

### DISCUSSION THREAD

**Discussion by PAKHAN8** [sebanc/brunch#1086] [General]
what is the best way of using windows / chromeos dualboot on real pixelbook COA ?
One day i want to install windows but do not know if dualboot possible with official chromeos images )
probably this brunch will solve the problem or there is othe way to use original image from google on the ntfs partition and grub ?

**Comment by simplyrajatgupta** [comment]
Given here
[https://github.com/sebanc/brunch/issues/1088]()

**Comment by kerta1n** [comment]
You should install [Mrchromebox.tech](https://mrchromebox.tech/#devices) UEFI, and then follow the instructions for dual booting [here](https://github.com/sebanc/brunch#dual-boot-chromeos-from-your-hdd).

**Reply by PAKHAN8** [reply]
Thank You !!!!!!
